<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditRootPath">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBoxAnonymous">
      <property name="text">
       <string>Allow anonymous login</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBoxOnlyOneIpAllowed">
      <property name="text">
       <string>Only one IP allowed</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QToolButton" name="toolButtonBrowse">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonShowDebugLog">
      <property name="text">
       <string>Debug Console</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBoxReadOnly">
      <property name="text">
       <string>Read-only</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonRestartServer">
      <property name="text">
       <string>Restart FTP Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPort">
      <property name="text">
       <string>21</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Root path:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditUserName"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxDisableLockScreen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Disable lock screen</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
